lib/i18n/tasks/scanners/model_scanner.rb

```diff
@@ -0,0 +1,38 @@
+module I18n::Tasks::Scanners
+  
+  # Parses model attribute lists as generated by the 'annotate' gem
+  #
+  class ModelScanner < FileScanner
+    include I18n::Tasks::Scanners::OccurrenceFromPosition
+    
+    def initialize(**args)
+      super
+      @patterns = [
+        [/\#  (?<a>[\w_]+)/, 'activerecord.attributes.%{model}.%{a}']
+      ]
+    end
+    
+    MODEL_NAME = /[^\/]+(?=\.rb)/
+    
+    def scan_file(path)
+      return [] unless path =~ /.*\/models\/.*.rb/
```

Comment:
Path matching should be handled during plugin registration for better performance.